frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.scss

```diff
@@ -34,3 +34,10 @@
       display: inline !important; //To override bootstrap help-block
   }
 }
+
+.ocs-storageClass-encryption__form-checkbox {
+  .pf-c-check__description {
+    grid-column: 1/3 !important; // To override patternfly description class
+    margin-bottom: var(--pf-global--spacer--sm);
+  }
```

Comment:
@Jayashree-panda Tagging you for awareness.

Reply:
@cloudbehl Thanks

Reply:
I feel let's not do this to genric components, then extension behavior is overridden. Lets update the value through onParamChange, let me try the first solution.